src/css.js

```diff
@@ -113,6 +113,18 @@ function getWidthOrHeight( elem, name, extra ) {
 		styles = getStyles( elem ),
 		isBorderBox = jQuery.css( elem, "boxSizing", false, styles ) === "border-box";
 
+	// Support: IE11+
+	// Fix for edge case in IE 11. See https://github.com/jquery/jquery/issues/1764
+	if ( !support.reliableFullscreenBox && window.top !== window.self &&
+		document.msFullscreenElement ) {
```

Comment:
@markelog 
>> Try testing that on TestSwarm then. :-)
>
> We can't right now, right now we do only ua-testing, which is bad substitution.

I meant that TestSwarm wraps everything in an additional wrapper iframe which changes results here.

> We can check this with e2e-tests. You might not remember this, but those ua-tests were created as temp solution.

I had an impression they exist so that we're sure we don't regress support tests results in our supported browsers. Many of those results are already implicitly tested via testing APIs that depend on them working.

Anyway, any such change is a plan reaching far further than this PR as it'd require much larger effort. We may add a test page in the integration folder but otherwise I don't think there's anything else to do in this PR at this point.